crates/router/src/core/verification.rs

```diff
@@ -42,8 +42,8 @@ pub async fn verify_merchant_creds_for_applepay(
             "application/json".to_string().into(),
         )])
         .set_body(RequestContent::Json(Box::new(request_body)))
-        .add_certificate(Some(cert_data))
-        .add_certificate_key(Some(key_data))
+        .add_certificate(Some(pii::Secret::new(cert_data)))
+        .add_certificate_key(Some(pii::Secret::new(key_data)))
```

Comment:
We have already imported `ExposeInterface` from the `masking` crate here, might as well import `Secret` from the `masking` crate directly. (Optional change by the way.)